App.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  Button,
+  Text,
+  View
+} from 'react-native';
+import { StackNavigator } from 'react-navigation';
+
+class AllActivitiesScreen extends Component {
+  static navigationOptions = {
+    title: 'All Activities',
+  };
+  render() {
+    const { navigate } = this.props.navigation;
+    return (
+      <View>
+        <Button
+          onPress={() => navigate('MyActivities')}
+          title="My Activities"
+        />
+        <Button
+          onPress={() => navigate('Map')}
+          title="Map"
+        />
+        <Button
+          onPress={() => navigate('Information')}
+          title="Information"
+        />
+      </View>
+    )
+  }
+}
+
+class MyActivitiesScreen extends Component {
+  static navigationOptions = {
+    title: 'My Activities',
+  };
+  render() {
+    const { navigate } = this.props.navigation;
```

Comment:
Nit: Change to `const { navigation: { navigate } } = this.props;` as mentioned in one of the above comments.